.github/workflows/ios-cloud-build.yml

```diff
@@ -47,7 +56,10 @@ jobs:
         bundle exec fastlane enterprise
```

Comment:
nemelo by se to tu upravit taky na tu Fastlane Beta lane? Nebo spatne chapu vyznam tech jednotlivych workflows? 🤔 
`bundle exec fastlane beta`